src/scenes/Projects/Overview/ProjectOverview.tsx

```diff
@@ -85,9 +92,17 @@ export const ProjectOverview: FC = () => {
             )}
           </Typography>
 
-          <Typography variant="h4">
-            {data ? 'Project Overview' : <Skeleton width="25%" />}
-          </Typography>
+          <div className={classes.subheader}>
+            <Typography variant="h4">
+              {data ? 'Project Overview' : <Skeleton width={200} />}
+            </Typography>
+
+            {data && (
+              <Typography color="textSecondary">
+                Updated {formatDate(data.project.modifiedAt)}
```

Comment:
modifiedAt is a DateTime and should use that formatter

Reply:
changed